base/server/python/pki/server/__init__.py

```diff
@@ -1221,7 +1221,8 @@ def load(self):
                     self.subsystems.append(subsystem)
 
     def load_external_certs(self, conf_file):
-        self.external_certs = PKIInstance.read_external_certs(conf_file)
+        external_certs = PKIInstance.read_external_certs(conf_file)
+        self.external_certs.extend([certs for certs in external_certs])
```

Comment:
This line tries to retrieve 1 element, convert that item into a list and then add it to the `self.external_certs` list. Instead, since the `read_external_cert()` method returns a list, how about simplifying a bit more?
````
self.external_certs.extend(PKIInstance.read_external_certs(conf_file))
````

(or) if you want the code to have more readability, something like this?

````
external_certs = PKIInstance.read_external_certs(conf_file)
for external_cert in external_certs:
    self.external_certs.append(external_cert)
````

Reply:
About first snip
```
self.external_certs.extend(PKIInstance.read_external_certs(conf_file))
```
`PKIInstance.read_external_certs(conf_file)` will return the `dict_values`, so appending it directly to the `self.external_certs` list is not good idea.

But second snip. It's good suggestion. I'll update it.

Reply:
IIUC, I think `PKIInstance.read_external_certs(conf_file)` returns a `list` as per the method [here](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/__init__.py#L1228)

Reply:
@SilleBille yes. You are pointing me to expected implementation of the the method. But it's working changes [here](https://github.com/dogtagpki/pki/blob/master/base/server/python/pki/server/__init__.py#L1244) and fails. Like below example:
```
>>> d = {'one':1, 'two': 2, 'three': 3}
>>> v = d.values()
>>> type(v)
<class 'dict_values'>
```